railgun/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -21,6 +21,6 @@ spec:
           name: https
       - name: manager
         args:
-        - "--health-probe-bind-address=:8081"
+        - "--health-probe-bind-address=:10254"
```

Comment:
No action required on this PR.

@shaneutt do we need this file at all?

Reply:
The RBAC proxy patch should be left alone yes, the controller manager uses this after it's patched onto the base deployment.

Reply:
Should it? AFAIK a patch that includes the current value _will_ override the health address if it's set to something else, and your revised pods will fail to come online unless you also update the probes.

I'm not actually sure why this arg (or `--leader-elect`) is present in the patch: including the metrics arg makes sense because it ensures that the manager's metrics listen is both only exposed within the Pod and on the port that the RBAC proxy sends to, but the RBAC proxy doesn't do anything with the health service, and it seems like `--health-probe-bind-address` should just be omitted from the patch/allowed to remain at whatever value it was originally.

Should we remove `--health-probe-bind-address` and `--leader-elect` from this patch entirely?